GameMap.py

```diff
@@ -0,0 +1,73 @@
+import curses
+
+
+class GameMap:
+    def __init__(self, player, tower):
+        self.player = player
+        self.tower = tower
+        # TODO: read events from file (?)
+        # Events: - pause
+        #         - inventory
+        #         - help
+        #         - move Player
+        self.events = {}
+        # Event log window
+        self.event_log = ""
+
+    # print the GameMap onto given screen
+    def print(self, window):
+        screen_size = window.getmaxyx()
+        game_map_win = curses.newwin(screen_size[0], screen_size[1], 0, 0)
+        # self.player.current_level.name (?)
+        game_map_win.addstr(1, 3, "Ebene {number}" + self.player + self.tower)
+        map_window = curses.newwin(int(screen_size[0] * 0.66) - 1,
+                                   int(screen_size[1] - 5), 2, 3)
+        map_window.border()
+        map_size = map_window.getmaxyx()
+        # self.player.current_level.print
+        map_window.addstr(
+            int(map_size[0] / 2), int(map_size[1] / 2) - 7, "{current level}")
```

Comment:
integer division funktioniert in python mit `//`, fände es gut wenn wir das verwenden

Reply:
Ah ok. Find ich persönlich etwas komisch zu lesen, aber damit es konsistent bleibt.